Hoarder Script Fails during package installation

### Please verify that you have read and understood the guidelines.

yes

### A clear and concise description of the issue.

> ✓ Updated Container OS
> Reading package lists... Done
> Building dependency tree... Done
> Reading state information... Done
> ca-certificates is already the newest version (20230311).
>  \Some packages could not be installed. This may mean that you have
> requested an impossible situation or if you are using the unstable
> distribution that some required packages have not yet been created
> or been moved out of Incoming.
> The following information may help to resolve the situation:
> 
> The following packages have unmet dependencies:
>  | chromium : Depends: libc++1-19 (>= 1:19.1.4) but it is not installable
>             Depends: libc++abi1-19 (>= 1:19.1.4) but it is not installable
>             Depends: libunwind-19 (>= 1:19.1.4) but it is not installable
>             Depends: chromium-common (= 131.0.6778.108-1~deb12u1) but it is not going to be installed
>             Recommends: chromium-sandbox but it is not going to be installed
> E: Unable to correct problems, you have held broken packages.
> 
> [ERROR] in line 17: exit code 0: while executing command $STD apt-get install -y g++ build-essential curl git sudo gnupg ca-certificates chromium mc
> root@pve:~#

### What settings are you currently utilizing?

- [X] Default Settings
- [X] Advanced Settings

### Which Linux distribution are you employing?

Debian 12

### If relevant, including screenshots or a code block can be helpful in clarifying the issue.

_No response_

### Please provide detailed steps to reproduce the issue.

Just kick off the script... : ]

--- a/install/hoarder-install.sh
+++ b/install/hoarder-install.sh
@@ -22,7 +22,8 @@ $STD apt-get install -y \
   sudo \
   gnupg \
   ca-certificates \
-  chromium \
+  chromium/stable \
+  chromium-common/stable \
   mc
 msg_ok "Installed Dependencies"
 

